baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -241,3 +345,127 @@ def build_message_handler(self) -> KombuMessageHandler:
 
     def build_health_checker(self, listener: socket.socket) -> StreamServer:
         return make_simple_healthchecker(listener, callback=self.health_check_fn)
+
+
+class KombuBatchQueueConsumerFactory(QueueConsumerFactory):
+    """Factory for running a :py:class:`~baseplate.server.queue_consumer.QueueConsumerServer` using Kombu with batch processing.
+
+    For simple cases where you just need a basic queue with all the default
+    parameters for your message broker, you can use `KombuQueueConsumerFactory.new`.
+
+    If you need more control, you can create the :py:class:`~kombu.Queue` s yourself and
+    use the constructor directly.
+    """
+
+    def __init__(
+        self,
+        baseplate: Baseplate,
+        name: str,
+        connection: kombu.Connection,
+        queues: Sequence[kombu.Queue],
+        handler_fn: BatchHandler,
+        error_handler_fn: Optional[BatchErrorHandler] = None,
+        health_check_fn: Optional[HealthcheckCallback] = None,
+        serializer: Optional[KombuSerializer] = None,
+        batch_size: int = 1,
+        batch_timeout: timedelta = timedelta(0, 1, 0, 0, 0, 0, 0),
```

Comment:
You can just write this as:

```suggestion
        batch_timeout: timedelta = timedelta(seconds=1),
```